packages/astro/src/i18n/index.ts

```diff
@@ -100,7 +114,7 @@ export function getLocaleRelativeUrlList({
 	format,
 	path,
 	prependWith,
-	normalizeLocale = false,
+	normalizeLocale = true,
```

Comment:
Just confirming that this in an internal API and this won't break any public APIs we've exposed.

Reply:
Yes, this is intentional. This is a bug! I will add a changeset for that